ddtrace/tracer/stats.go

```diff
@@ -166,7 +157,7 @@ func (c *concentrator) Stop() {
 	c.wg.Wait()
 }
 
-func (c *concentrator) flush(timenow time.Time) statsPayload {
+func (c *concentrator) flush(timenow time.Time, force bool) {
```

Comment:
```go
// flush flushes all the stats buckets with the given timestamp. The current bucket is only included if 
// includeCurrent is true, such as during shutdown.
func (c *concentrator) flush(timenow time.Time, includeCurrent bool) {
```
I think `force` is not a good word nor approach to anything in life 🙂 And while boolean args are considered a smell (their purpose is ambiguous when looking at the call), we can live with it here because it's unexported and we can do a small trick to self-document it:

```go
const (
    withCurrentBucket    = true
    withoutCurrentBucket = false
)
```

Then, when calling flush, use the two constants.

Reply:
👍 totally agreed!